docs/reference/api/docker_remote_api_v1.24.md

```diff
@@ -3501,6 +3501,89 @@ Return low-level information on the node `id`
 -   **404** – no such node
 -   **500** – server error
 
+### Remove a node
+
+
+`DELETE /nodes/<id>`
+
+Remove a node [`id`] from the Swarm.
```

Comment:
I think it's ok to only mention `ID`, and fix the code separately (keeping it an "undocumented feature" for now), otherwise we need to go through a deprecation process if we remove the option of using full name or ID prefix.

Code should be fixed though if we don't want to support it